pkg/ibmcloud/types.go

```diff
@@ -656,3 +656,68 @@ type GithubIssueClusterRequest struct {
 	ErrorCount int    `json:"errorCount"`
 	Regions    string `json:"regions"`
 }
+
+type UserInviteList struct {
+	Users []UserInvite `json:"users,omitempty"`
+	Resources []UserInvite `json:"resources,omitempty"`
+}
+// type UserInvite struct {
+// 	Email       string `json:"email"`
+// 	AccountRole string `json:"account_role,omitempty"`
+// }
+
+// type UserInviteResponseList struct {
+// 	Resources []UserInviteResponse `json:"resources"`
+// }
+
+type UserInvite struct {
+	ID    string `json:"id,omitempty"`
+	AccountRole string `json:"account_role,omitempty"`
+	Email string `json:"email"`
+	State string `json:"state,omitempty"`
+}
+
+type MemberList struct {
+	Members []Member `json:"members"`
+}
+
+type Member struct {
+	IamID       string     `json:"iam_id"`
+	Type        string     `json:"type"`
+	CreatedAt   *time.Time `json:"created_at,omitempty"`
+	CreatedByID string     `json:"created_by_id,omitempty"`
+	StatusCode  int        `json:"status_code,omitempty"`
+}
+
+type Attribute struct {
+	Name     string `json:"name"`
+	Value    string `json:"value"`
+	// Operator string `json:"operator,omitempty"`
+}
+
+type Policy struct {
+	Type        string      `json:"type"`
+	Description string      `json:"description"`
+	Subjects    []AttributeList  `json:"subjects"`
+	Roles       []Role     `json:"roles"`
+	Resources   []AttributeList `json:"resources"`
+}
+type PolicyResponse struct {
```

Comment:
looks like a go format issue. 

if you are using vscode `cmd + shift + p` and format this file. 

in goland i think you need to enable formatting short cut or it might me `opt + cmd + L` 

You can also just use the `go fmt` command to do it from terminal